### ISSUE TITLE
Can't locate promises from {"type":"DENO.LAND URL ...}

### ISSUE THREAD

**Issue by waptik** [garronej/denoify#60]
So i am trying to denoify(v1.0.2) a forked project at https://github.com/waptik/epubgen and i'm getting the following error:
```sh
Denoify is reading sources files from lib
Deno distribution will be generated at deno_dist
/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/bin/denoify.js:7
    throw error;
    ^

Error: Can't locate promises from {"type":"DENO.LAND URL","isStd":true,"baseUrlWithoutBranch":"https://deno.land/std","pathToIndex":"node/fs.ts"}
    at /home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/lib/resolveNodeModuleToDenoModule/resolveNodeModuleToDenoModule.js:374:19
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async callee (/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/tools/addCache.js:19:33)
    at async denoifyImportExportStatement (/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/lib/denoifyImportExportStatement.js:77:26)
    at async /home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/lib/denoifySingleFile.js:91:60
    at async Promise.all (index 0)
    at async replaceAsync (/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/tools/replaceAsync.js:12:18)
    at async denoifySingleFile (/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/lib/denoifySingleFile.js:90:38)
    at async transformSourceCodeString (/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/lib/denoify.js:114:39)
    at async transformCodebase (/home/waptik/code/node/epubgen/node_modules/.pnpm/denoify@1.0.2/node_modules/denoify/tools/transformCodebase.js:13:49)



```

**Comment by garronej** [comment]
Sorry about this error, I'll have a look

**Comment by waptik** [comment]
> Sorry about this error, I'll have a look

Ok, thanks.

**Comment by garronej** [comment]
@waptik, I see the problem, fixing...

**Comment by garronej** [comment]
@waptik it's fixed in v1.0.3.  
There are many NPM dependencies though, you'll get things like:  
```typescript
import { compile, selectAll, selectOne } from 'npm:css-select@4.1.3';
import render from 'npm:dom-serializer@1.3.2';
import { Element, Node } from 'npm:domhandler@4.2.2';
import { removeElement } from 'npm:domutils@2.8.0';
import { parseDocument } from 'npm:htmlparser2@7.1.2';
import type { EPub } from '../index.ts';
import { allowedAttributes, allowedXhtml11Tags } from './constants.ts';
import type { CB } from './html.ts';
```  

NPM support is still experimental in Deno but in a few weeks it should work.  

Thanks for reporting!
